dracclient/resources/raid.py

```diff
@@ -82,7 +82,7 @@
     ['id', 'description', 'controller', 'manufacturer', 'model', 'media_type',
      'interface_type', 'size_mb', 'free_size_mb', 'serial_number',
      'firmware_version', 'status', 'raid_status', 'sas_address',
-     'device_protocol'])
+     'device_protocol', 'drac_bus_number'])
```

Comment:
Could you rename "drac_bus_number" to "bus" so that it's consistent with how we report the PCI bus number for controllers?  See:
https://github.com/openstack/python-dracclient/blob/master/dracclient/resources/raid.py#L144

Reply:
Sure.. Will do